pkg/odo/genericclioptions/context.go

```diff
@@ -468,16 +468,18 @@ func newDevfileContext(command *cobra.Command) *Context {
 
 	envInfo, err := getValidEnvinfo(command)
 	if err != nil {
-		util.LogErrorAndExit(err, "")
+		util.LogErrorAndExit(err, "unable to retrieve configuration information")
```

Comment:
Should we clarify which configuration info we couldn't retrieve?

Reply:
we cannot as this error should be related to file read, so its all or nothing.